cpg_utils/cloudpath_hail_az.py

```diff
@@ -4,26 +4,90 @@
 """
 
 import re
-from typing import Union, Optional
+import os
+import json
+import logging
+from typing import Union, Optional, Any, Callable
 from urllib.parse import urlparse
-
+import mimetypes
+from azure.identity import DefaultAzureCredential, ClientSecretCredential
+from azure.storage.blob import BlobServiceClient
+#import azure.storage.blob
```

Comment:
vestigial?

Reply:
the azure.* imports are used in HailAzureBlobClient.__init__(), mimetypes is only used to provide a default value for constructor's parameter `content_type_method` which gets passed to the superclass constructor.